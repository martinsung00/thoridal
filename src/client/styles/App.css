.divider {
  position: relative;
  height: 2em;
  top: 9em;
  left: 3.7em;
  width: 90%;
  position: relative;
  text-align: center;
  border-top: 0.7px solid black;
  border-bottom: 0.7px solid black;
}

.title {
  position: relative;
  text-align: center;
  top: 6em;
  font-family: Helvetica, Arial, sans-serif;
  font-size: 28px;
}
