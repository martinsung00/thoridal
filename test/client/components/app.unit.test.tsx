import React from "react";
import renderer from "react-test-renderer";
import App from "../../../src/client/components/App";

describe("App Tests", function () {
  it("should render hello world", function () {
    const container = renderer.create(<App />).toJSON();
    expect(container).toMatchSnapshot();
  });
});
