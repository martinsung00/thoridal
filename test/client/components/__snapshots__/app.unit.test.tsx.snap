// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App Tests Component Did Mount should make an api call to retrieve all trades 1`] = `ReactWrapper {}`;

exports[`App Tests Component Did Mount should set history to false if the result is empty 1`] = `ReactWrapper {}`;

exports[`App Tests Handle Delete Method should delete the desired trade 1`] = `ReactWrapper {}`;

exports[`App Tests Handle Error Method should set state to errored 1`] = `ShallowWrapper {}`;

exports[`App Tests should render header and menu 1`] = `
<div>
  <div>
    <div>
      <div>
        THORIDAL
      </div>
      <img
        alt="Main Logo"
        src={[Function]}
      />
    </div>
  </div>
  <div>
    <div>
      <div
        id="menu-add"
        onClick={[Function]}
      >
        <div>
          Write a Trade
        </div>
        <div />
        <div />
        <div />
      </div>
    </div>
    <div>
      <div
        id="menu-search"
        onClick={[Function]}
      >
        <div>
          Find a Trade
        </div>
        <div />
        <div />
      </div>
    </div>
    <div>
      <div
        id="menu-delete"
        onClick={[Function]}
      >
        <div>
          Delete a Trade
        </div>
        <div />
        <div>
          Ã—
        </div>
      </div>
    </div>
    <div>
      <div
        id="menu-update"
        onClick={[Function]}
      >
        <div>
          Update a Trade
        </div>
        <div>
          <div />
          <div />
        </div>
        <div>
          <div />
          <div />
        </div>
        <div>
          <div />
          <div />
        </div>
      </div>
    </div>
  </div>
  <br />
  <div>
    <div>
      Weekly Statistics
    </div>
    <div>
      <label>
        NET GAIN: 
        <div>
          +
          0
        </div>
      </label>
      <div />
      <label>
        TOTAL TRADES:
         
        <div>
          0
        </div>
      </label>
      <div />
      <div />
      <div />
      <label>
        Total Completed Trades:
         
        <div>
          0
        </div>
      </label>
      <div />
      <label>
        Total Pending Trades:
         
        <div>
          0
        </div>
      </label>
    </div>
    <div>
      <label>
        AVG GAIN PER DAY:
         
        <div>
          $ 
          0
        </div>
      </label>
      <div />
      <label>
        AVG TRADE PER DAY:
         
        <div>
          0
        </div>
      </label>
    </div>
  </div>
  <br />
  <div>
    Recent Trade(s)
  </div>
  <br />
  <div>
    <br />
    <div>
      <div
        style={
          Object {
            "marginTop": "-4em",
          }
        }
      />
    </div>
  </div>
</div>
`;
